scripts/all-tests.sh

```diff
@@ -80,6 +80,7 @@ case "$TEST_SUITE" in
         # Run quality task. Pass in the 'fail-under' percentage to diff-quality
         paver run_quality -p 100
 
+        paver run_complexity > code_complexity.log || echo "Unable to calculate code complexity. Ignoring error."
```

Comment:
Along those lines, have you looked into the `--xml` option?  The documentation suggests that it works with a Jenkins plugin to report them.